src/overcooked_ai_py/data/testing/test_display_phi/exected.txt

```diff
@@ -0,0 +1,309 @@
+X       X       P       X       X       
```

Comment:
The name of this file seems like a typo? :)

Reply:
fixed :P